config/development.json

```diff
@@ -12,5 +12,6 @@
         "username": "igvf",
         "password": "igvf"
     }
-  }
+  },
+  "cluster": ["0.0.0.0"]
```

Comment:
just an example of the format. In production, this will have all IPs of the nodes of the cluster